From ddb71909fd3be5af487471895cc4854eee339375 Mon Sep 17 00:00:00 2001
From: franciscofranco <franciscofranco.1990@gmail.com>
Date: Fri, 6 Dec 2013 17:29:22 +0000
Subject: [PATCH 202/590] Revert "net: wireless: bcmdhd: Set
 MAX_DTIM_ALLOWED_INTERVAL to 600"

This reverts commit 06e9b4e2568daeb12f39ed69df9552e8f1117533.

Signed-off-by: Simarpreet Singh <simar@linux.com>
---
 drivers/net/wireless/bcmdhd/dhd.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/bcmdhd/dhd.h b/drivers/net/wireless/bcmdhd/dhd.h
index 67e82a2..a6f587e 100644
--- a/drivers/net/wireless/bcmdhd/dhd.h
+++ b/drivers/net/wireless/bcmdhd/dhd.h
@@ -792,7 +792,7 @@ extern uint dhd_force_tx_queueing;
 #endif /* WLTDLS */
 
 #define MAX_DTIM_SKIP_BEACON_INTERVAL	100 /* max allowed associated AP beacon for DTIM skip */
-#define MAX_DTIM_ALLOWED_INTERVAL 600 /* max allowed total beacon interval for DTIM skip */
+#define MAX_DTIM_ALLOWED_INTERVAL 300 /* max allowed total beacon interval for DTIM skip */
 #define NO_DTIM_SKIP 1
 
 #ifdef SDTEST
-- 
1.7.9.5

